google/cloud/opentelemetry/internal/monitored_resource.cc

```diff
@@ -74,6 +74,8 @@ class MonitoredResourceProvider {
         mr.labels[kv.first] = AsString(attributes.at(*found));
       } else if (kv.second.fallback) {
         mr.labels[kv.first] = *kv.second.fallback;
+      } else {
+        mr.labels[kv.first] = "";
```

Comment:
So it seems like we always fallback. Sometimes to a value like `"global"`, but at least to `""`.

optional nit: It seems slightly cleaner to make `fallback` a `std::string`:

https://github.com/googleapis/google-cloud-cpp/blob/644efe9cf17e4a29397aa326e9fce98e4792e3e8/google/cloud/opentelemetry/internal/monitored_resource.cc#L55

and then have this code just be:

```cc
} else {
  mr.labels[kv.first] = kv.second.fallback;
}
```

(If this doesn't work for whatever reason, just let me know, and I can look into cleaning it up in a future PR)

Reply:
That does seem cleaner, I’ll change the fallback to default to empty string.

Reply:
Thanks!